package PassSystemConfig;

message PassItem
{
	required uint32 	id			 	= 1;
	required uint32		terrain			= 2;//地形
	required uint32		cost_food		= 3;//消耗粮食
	required uint32		unlock_lvl		= 4;//解锁等级
	required bool		power_switch	= 5;//战法开关
	required bool		is_record_cnt	= 6;//是否记录闯关次数
	repeated uint32		npcs			= 7;//npc队列
}

message System
{
 	repeated PassItem	all_upc_pass 	= 1;
}